src/main/java/org/example/App.java

```diff
@@ -15,7 +15,16 @@ public static void main( String[] args ) {
      * @param str Array of words which must be sorted
      * @return List of sorted words
      */
-    public static List<String> sortList(String[] str){
-        return Arrays.asList(str).stream().sorted().collect(Collectors.toList());
+    public static List<String> sortList(String[] str) throws IllegalArgumentException{
+        if(str.length == 0){
+            throw new IllegalArgumentException("No parameter passed!");
+        }
+        else if(str.length == 1) {
+            return Arrays.asList("You passed only one parameter, it must be two or more!");
```

Comment:
Why don't throw an exception here as well, as you've done above? Why do we need the list here?

I see that by doing so you want to user let know that something went wrong. Please follow my comment above and split the responsibility so that you will be able to trow and exception and catch in the catch block of the main method.